netbout-web/src/main/java/com/netbout/rest/bout/TkAttach.java

```diff
@@ -54,6 +55,7 @@
 import org.takes.facets.forward.RsForward;
 import org.takes.rq.RqHeaders;
 import org.takes.rq.RqMultipart;
+import org.takes.rq.RqMultipart.Smart;
```

Comment:
@jhyle don't use such imports, the correct class name should be `RqMultipart.Smart`, not just `Smart`